homeassistant/components/elgato/strings.json

```diff
@@ -1,17 +1,17 @@
 {
   "config": {
-    "flow_title": "Elgato Key Light: {serial_number}",
+    "flow_title": "Elgato Light: {serial_number}",
     "step": {
       "user": {
-        "description": "Set up your Elgato Key Light to integrate with Home Assistant.",
+        "description": "Set up your Elgato Light to integrate with Home Assistant.",
```

Comment:
Side note in general for HA:
I always wonder if we should even have descriptions like these, that are not really adding much value since we have already integration name in the title.

Reply:
I guess, dunno, I think describing the action one is taking, in general, is a good thing. 🤷